docs/getting-started.md

```diff
@@ -70,16 +70,16 @@ guide will not use the [`jailer`](../src/jailer/).
 ARCH="$(uname -m)"
 
 # Download a linux kernel binary
-wget https://s3.amazonaws.com/spec.ccfc.min/img/quickstart_guide/${ARCH}/kernels/vmlinux.bin
+wget https://s3.amazonaws.com/spec.ccfc.min/firecracker-ci/v1.5/${ARCH}/vmlinux-5.10.186
```

Comment:
Just out of curiosity, are we going to regenerate CI artifacts and update these links every minor release? If yes, can we include this update in the release automation?